grakn/concept/answer/concept_map.py

```diff
@@ -23,12 +23,12 @@
 
 from grakn.common.exception import GraknClientException
 from grakn.concept.proto import concept_proto_reader
-from grakn.concept.answer.answer import Answer
 from grakn.concept.concept import Concept
 
 
-class ConceptMap(Answer):
+class ConceptMap:
 
+    _CONCEPT_MAP = "concept_map"
```

Comment:
i don't think you're actually using this anywhere

Reply:
Fixed in https://github.com/graknlabs/client-python/pull/148/commits/6406a2970bd49e9458fd226d93beafafdf37c449